utils_linux.go

```diff
@@ -240,7 +241,15 @@ func (r *runner) run(config *specs.Process) (int, error) {
 			return -1, err
 		}
 	}
-	if r.detach {
+	if !r.create {
+		if err := process.Signal(syscall.SIGCONT); err != nil {
+			r.terminate(process)
+			r.destroy()
+			tty.Close()
+			return -1, err
+		}
+	}
+	if r.detach || r.create {
```

Comment:
On Tue, May 17, 2016 at 01:30:08PM -0700, W. Trevor King wrote:
> wait(1p) is a fairly convenient way to do that from the shell…

Actually, this doesn't work:

  $ (sh -c 'sleep 60 &') &
  $ ps -u wking | grep sleep | grep -v grep
  22881 pts/13   00:00:00 sleep
  wking@ullr ~ $ wait 22881
  bash: wait: pid 22881 is not a child of this shell

And waitpid(2) is talking about child processes, not ancestor
processes.  So I'm back to wondering how you collect the container
exit status when the ‘create’ process exits early.